backend/tests/endpoints/test_room.py

```diff
@@ -1,4 +1,127 @@
+import pytest
 from fastapi.testclient import TestClient
 from main import app
 
 client = TestClient(app)
+
+join_room_test_url = "api/v1/org/619org/rooms/619Chrm1/members/619mem1"
+test_payload = {"619mem3": {"role": "member", "starred": False, "closed": False}}
+fake_core_room_data = {
```

Comment:
I think this name is too generic. Consider explicitly renaming this so there's no potential conflict with other peoples tests. 

Except you're creating a fake_core_room_data that is reuseable for everyone's test requirements

Reply:
depends on what the person is testing sha, I just used the minimal K-V pairs required for my test cases. But definitely it can be made to replicate the core room object by adding all the possible fields so that other tests can use it, only making minor modifications to the fields in their tests where necessary depending on what they want to test.
As for the name, the idea is for it to be generic, I will update the fields to reflect that.

Reply:
@pauline-banye nah, I don't think that is necessary, that will mean creating several fake room data for every test case, that will make the entire file messy. If you check the tests cases I covered you will find that I tested for channels, dm and group_dm without having to create different fake room data for each of them. 

So it is not an issue and like I said earlier, the fake room data fields can be modified in each test case to suit what the person wants to test for.